modules/core/src/main/scala/org/tessellation/Main.scala

```diff
@@ -106,6 +108,9 @@ object Main
 
               Signed.forAsyncKryo[IO, GlobalSnapshot](genesis, keyPair).flatMap { signedGenesis =>
                 storages.globalSnapshot.prepend(signedGenesis) >>
+                  services.collateral
+                    .hasCollateral(sdk.nodeId)
+                    .flatMap(OwnCollateralNotSatisfied.raiseError[IO, Unit].unlessA) >>
```

Comment:
does it stop the process and node stops immediately?

Reply:
Yes, I have checked it locally.